sdk/analysisservices/Azure.ResourceManager.Analysis/CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ New design of track 2 initial commit.
 - Corrected the format of all `ETag` type properties / parameters.
 - Corrected the format of all `AzureLocation` type properties / parameters.
 - Corrected the type of property `ResourceType` of ExistingResourceSkuDetails from `String` to `ResourceType`.
+- Corrected the namespace of this project from `Azure.Management.AnalysisServices` to `Azure.ResourceManager.Analysis`
+- Corrected the prefix `AnalysisServices` of a subset of classes in this project from `AnalysisServices` to `Analysis`
```

Comment:
Since we haven't released this package, I think we don't need these two items. Just update the package name section.

Reply:
fixed